web/templates/settings/notification_settings.hbs

```diff
@@ -106,7 +106,11 @@
             </h3>
             {{> settings_save_discard_widget section_name="desktop-message-settings" show_only_indicator=(not for_realm_settings) }}
         </div>
-
+        <div class="tip desktop-setting-notications">
```

Comment:
Do we want to show the banner in "Default user settings" panel as well?

Reply:
No, I don't think it will make sense to show a banner there based on the current user's personal state.